cdk/bin/cdk.ts

```diff
@@ -1,7 +1,12 @@
 import "source-map-support/register";
-import { App } from "aws-cdk-lib";
+import { GuRoot } from "@guardian/cdk/lib/constructs/root";
+import type { App } from "aws-cdk-lib";
 import { GoogleSearchIndexChecker } from "../lib/google-search-index-checker";
-import {GuRootExperimental} from "@guardian/cdk/lib/experimental/constructs/root";
 
-const app: App = new GuRootExperimental();
-new GoogleSearchIndexChecker(app, "GoogleSearchIndexChecker-PROD", { stack: "ophan", stage: "PROD", env: {region: "eu-west-1"} });
+const app: App = new GuRoot();
+new GoogleSearchIndexChecker(app, "GoogleSearchIndexChecker-PROD", {
+    stack: "ophan",
+    stage: "PROD",
+    env: { region: "eu-west-1" },
+    withBackup: true
```

Comment:
This is turning on the backups: https://github.com/guardian/cdk/releases/tag/v52.2.0